bundles/automation/org.eclipse.smarthome.automation.core/src/main/java/org/eclipse/smarthome/automation/internal/core/RuleEngine.java

```diff
@@ -150,6 +152,7 @@
     public RuleEngine(BundleContext bc) {
         this.bc = bc;
         logger = LoggerFactory.getLogger(getClass());
+        context = new HashMap<String, Object>();
```

Comment:
Only one context for the whole rule engine? Doesn't this fill up with values from all the different rules?